pkg/ui/src/views/cluster/containers/nodeOverview/index.tsx

```diff
@@ -180,9 +184,11 @@ export default connect(
     return {
       node: currentNode(state, ownProps),
       nodesSummary: nodesSummarySelector(state),
+      nodesSummaryValid: selectNodesSummaryValid(state),
```

Comment:
it looks like this isn't actually used anywhere?

Reply:
It needs to be on there or it won't actually refresh when the status becomes invalid.

Reply:
how does that work if there are no references to it?

Reply:
You're looking at the reference right there; the object returned from mapStateToProps is what react-redux uses to determine if a component re-renders or not.

Reply:
That is not at all clear from reading the code.

Reply:
That is how all of the automatic refreshes work on every page.

I realize it is confusing, #18741 was supposed to address this (it's called out specifically in that issue) but has been sidelined for the foreseeable future.